releasenotes/notes/submodule-update-a2f6c5e9cb597a99.yaml

```diff
@@ -0,0 +1,33 @@
+---
+critical:
+  - |
+    From this release on, a rcbops fork is used for the openstack-ansible
+    submodule. Due to this change, a pre-existing checkout of rpc-openstack will
+    require a submodule sync to ensure that the subodule is pointing to the
+    right repository. Failing to keep the submodule in sync will result in some
+    unexpected behavior, primarily you will be missing any code updates that
+    have been pushed up to the forked repository. To resolve this execute the
+    following:
+
+    .. code-block: shell
+      $ git submodule status
+      $ git submodule update --init
+      $ git submodule sync
+      $ git submodule status
```

Comment:
The status usage here isn't very useful. Perhaps something like this is more useful given the circumstance?

```
$ git config --list | grep \^submodule
submodule.openstack-ansible.active=true
submodule.openstack-ansible.url=https://git.openstack.org/openstack/openstack-ansible

$ git submodule update --init
Submodule path 'openstack-ansible': checked out '5644b36dbd74352d7503f809760a0684f0a84a9d'

$ git submodule sync
Synchronizing submodule url for 'openstack-ansible'

$ git config --list | grep \^submodule
submodule.openstack-ansible.active=true
submodule.openstack-ansible.url=https://github.com/rcbops/openstack-ansible
```

Reply:
So this brings up a good point, there are two use cases here:

1) The submodule is still pointing to the original openstack repo
2) The submodule is pointing to the forked rcbops submodule but the SHA is out of date

The steps for both are the same but I think the output for the former will be helpful.

Reply:
Ah, but the first use-case is not the subject of the release note. Teaching people how to update the submodule is not the goal of the release note - that should already be in the documentation because that's a normal minor upgrade procedure. The goal of the release note is to inform people of an unusual situation, specifically that the submodule requires a sync to work when updating from a previous release to this release.